spec/claCheckSpec.js

```diff
@@ -186,7 +189,17 @@ describe('CLA check github action Module for Oppia', () => {
         ' Welcome to Oppia! Could you please ' +
         'follow the instructions ' + LINK_RESULT +
         " and sign the CLA Sheet to get started? You'll need to do " +
-        'this before we can accept your PR. Thanks!');
+        "this before we can accept your PR. Once you\'re done," +
+        ' please reopen the PR. Thanks!');
+
+      expect(octokit.issues.update).toHaveBeenCalled();
+
+      expect(octokit.issues.update).toHaveBeenCalledWith({
```

Comment:
I think just `toHaveBeenCalledWith` is enough, since it checks for some call implicitely.

Reply:
ditto below

Reply:
Done